% Original Template Author: Rishi Shah

% Document Configuration
\documentclass{resume}
\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry}
\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}
\name{Jack Gisby}
\address{Address: Available on Request \\ DOB: 28/02/1998}
\address{Tel: (+44)7748 559964 \\ Email: jackgisby@gmail.com \\ Github: https://github.com/jackgisby}

% Document Content 
\begin{document}
\begin{rSection}{Research Interests}
PhD candidate at Imperial College London with a keen interest in data-driven research and the omics technologies. I have the aspiration to pursue a career in biological research, exploiting the vast datasets produced by modern biology to answer fundamental research questions.

\end{rSection}
\begin{rSection}{Education}

\smallskip
{\bf Imperial College London} \hfill {\em July 2020 - August 2023} 
\\ PhD Candidate - Department of Immunology and Inflammation

\smallskip
\item Thesis: Using Multi-Omics to Understand Inflammatory Disease
\item Supervised by Dr James Peters and Dr Jacques Behmoaras 
\item Assessed by Professor Marc Chadeau and Dr Jessica Strid

\medskip

{\bf University of Birmingham} \hfill {\em September 2016 - June 2020} 
\\ Biochemistry with Professional Placement (MSci) - First Class

\smallskip
\item Dissertation: A Prior Knowledge-Based Computational Workflow for \textit{de novo} Structural Elucidation of Small Molecules in Mass Spectrometry Metabolomics

\end{rSection}

% \begin{rSection}{Awards and grants}
% \end{rSection}

\begin{rSection}{Selected Publications}

\textbf{The widespread nature of Pack-TYPE transposons reveals their importance for plant genome evolution} \\
First author \hfill  Manuscript in progress

\smallskip
\item Designed an algorithm for the specific annotation of repetitive elements that capture chromosomal DNA; implemented this as an R package available as part of the Bioconductor project.
\item Mined open source genetic data to demonstrate the abundance of these elements for multiple superfamilies and genomes. Found that recent insertions of these elements have impacted the evolution of genes. \\

\textbf{Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death} \\
First author \hfill  \textit{eLife} 2021 - doi:10.7554/eLife.64827

\smallskip
\item Lead the analysis of a high-dimensional proteomics (Olink) dataset with a complex cohort consisting of repeated measurements at inconsistent time points.
\item Applied linear mixed models and joint models to identify key proteins that changed over time following COVID-19 symptom onset.
\item Utilised supervised learning algorithms to identify biomarkers of severe disease.\\
 
\textbf{Mendelian randomisation identifies alternative splicing of the FAS death receptor as a mediator of severe COVID-19} \\
Equal first contributor \hfill  medRxiv 2021 - doi:10.1101/2021.04.01.21254789

\smallskip
\item Developed a pipeline to apply two sample Mendelian Randomisation comparing COVID-19 GWAS to Olink pQTLs. 
\item Used fine mapping and colocalisation tools in tandem with public data repositories to investigate causal variants of disease. \\

\textbf{Plasma Lectin Pathway Complement Proteins in Patients With COVID-19 and Renal Disease} \\
Equal first contributor \hfill  \textit{Frontiers in Immunology} 2021 - doi:10.3389/fimmu.2021.671052

\smallskip
\item Provided statistical support for the analysis of a repeated measures study design.
\item Generated data visualisations to investigate biological hypotheses. \\

\end{rSection}

\begin{rSection}{Conferences and presentations}

\item \textbf{Poster - UK-CIC Immunology 2021} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death

\item \textbf{Presentation - Longitudinal Studies 2021, Wellcome Genome Campus} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death

\item \textbf{Presentation - Centre for Inflammatory Disease, Imperial College London} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death

\end{rSection}

\begin{rSection}{Additional research experience}

\textbf{A Prior Knowledge-Based Computational Workflow for \textit{de novo} Structural Elucidation of Small Molecules in Mass Spectrometry Metabolomics} \\
Dissertation project, Dr Ralf Weber \hfill  \textit{October 2019 - June 2020}

\smallskip
\item Developed and optimised a Python package, \textit{Metaboblend}, to propose candidate molecules using tandem mass spectra. 
\item Processed large, high-dimensional mass-spectrometry datasets for the validation of \textit{Metaboblend}. 
\item Used Python, SQLite and Bash to implement the \textit{Metaboblend} workflow on a Linux high-performance computing cluster for the generation of large structure databases. \\

\textbf{Development of an Immunoturbidimetric Assay for Serum Amyloid A for an Automated Clinical Analyser} \\
Industrial Placement, The Binding Site \hfill  \textit{August 2018 - August 2019}

\smallskip
\item Developed assays for inflammatory biomarkers, such as Serum Amyloid A, for use in clinical laboratories.
\item Interacted with, and regularly delivered presentations to, researchers from diverse fields and non-scientific staff to ensure the timely completion of my development project. 
\item Took responsibility for planning and carrying out experiments and statistical analyses in a research environment.

\end{rSection}

\begin{rSection}{Teaching experience}

\item \textbf{Code Club Volunteer} - developed my teaching abilities and provided help with coding skills to 9-13 year-olds as part of the Code Club voluntary initiative.

\end{rSection}

\begin{rSection}{Specific Computational Skills}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{6ex}} l }
Operating Systems \ & Windows, Ubuntu \\
Programming Languages \ & Python, R, SQLite, Bash, working knowledge of C++ \\
Packages \ & Bioconductor, ggplot2, scikit-learn, tensorflow \\
Other Tools \ & Git, Linux HPC \\
General Software \ & MS Office, Latex  \\
\end{tabular}

\end{rSection}
\begin{rSection}{Additional Courses}

\item Systems biology: From large datasets to biological insight (1 week) by Wellcome Connecting Science and EMBL-EBI
\item Introduction to Assessment and Feedback by Imperial College London
\item Introduction to Teaching and Learning by Imperial College London
\item Profiling and optimisation in Python by Imperial College London
\item Computational Systems Biology: Deep Learning in the Life Sciences by MITx
\item Software Development with C++ (1 week) by University of Birmingham Research Computing
\item The Biostars Handbook: Bioinformatics Data Analysis by Istvan Albert
\item Introduction to Computer Science and Programming Using Python by MITx
\item Introduction to Computational Thinking and Data Science by MITx
\item Data Analysis for Life Sciences and Genomics Data Analysis by HarvardX

% \begin{rSection}{Additional publications}
% \end{rSection}

\end{rSection}
\end{document}
